packages/echo/src/handler.ts

```diff
@@ -23,16 +23,18 @@ import {
   SignatureNotFoundError,
   SigningKeyNotFoundError,
 } from './errors';
-import { Awaitable } from './types';
+import { Awaitable, DiscoverWorkflowOutput } from './types';
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export interface ServeHandlerOptions {
-  client: Echo;
+  client?: Client;
+  workflow: Array<DiscoverWorkflowOutput>;
 }
 
-interface IEchoRequestHandlerOptions<Input extends any[] = any[], Output = any> extends ServeHandlerOptions {
+interface INovuRequestHandlerOptions<Input extends any[] = any[], Output = any> extends ServeHandlerOptions {
   frameworkName: string;
-  client: Echo;
+  client?: Client;
+  workflow: Array<DiscoverWorkflowOutput>;
```

Comment:
I've seen this in the discussion in slack and in the DX guide. any particular reason why we are using a singular for a property that is containing an array?

Reply:
Good catch, that was a typo! It should be plural `workflows`

Reply:
```suggestion
  workflows: Array<DiscoverWorkflowOutput>;
```